_cloud/__init__.py

```diff
@@ -9,7 +9,11 @@
 
 
 class Module(object):
-    """The magical API around downloading / installing /Â importing things."""
+    """The magical API around downloading / installing / importing things."""
+    
+    _zip = None
+    metadata = None
```

Comment:
I don't think pylint is something we're developing against. I'm using pep8 and pyflakes. Let me take a look when I get to a laptop in a few hours, though.